scheduler/src/cook/mesos/util.clj

```diff
@@ -43,25 +44,12 @@
       (.expireAfterAccess 2 TimeUnit/HOURS)
```

Comment:
Should we also move `new-cache`?

Reply:
I view cook.cache as 'too global' to put the factory method with a full config. None of the rest of the code uses any configuration and I could envision wanting that to be different in different use-cases.